src/Schema/Keywords/MultipleOf.php

```diff
@@ -18,18 +18,21 @@ class MultipleOf extends BaseKeyword
      *
      * @param mixed     $data
      * @param int|float $multipleOf
+     *
+     * @throws ValidationKeywordFailed
      */
     public function validate($data, $multipleOf) : void
     {
         try {
             Validator::numeric()->assert($data);
             Validator::numeric()->positive()->assert($multipleOf);
 
-            if ($data % $multipleOf) {
-                throw new Exception(sprintf('Division by %d did not resulted in integer', $multipleOf));
+            $value = $data / $multipleOf;
+            if ((int) $value !== $value) {
```

Comment:
https://swagger.io/docs/specification/data-models/data-types/#multipleOf

`multipleOf` could be used against floats, so I patched this a bit while inspecting validator